Makefile

```diff
@@ -205,11 +205,18 @@ endif
 # ------------
 
 .PHONY: verify-deps
-verify-deps:
+verify-deps: verify-go-deps verify-src-deps
```

Comment:
I don't think there is no benefit this target hierarchical. Let's just keep all `verify-*` scripts directly under `verify`